roles/backpack/templates/backpack.yml

```diff
@@ -46,5 +50,5 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
-      serviceAccountName: {{ metadata_sa | default('default') }}
+      serviceAccountName: {{ metadata.sa | default('default') }}
```

Comment:
The only nit I have... sa makes sense to us, and yeah, it is defaulted... but given it is defaulted and we don't always have to pass it. Let's make it more obvious and call it `metadata.serviceaccount`

Reply:
Good call. Updated